keymaps/omnisharp-atom.cson

```diff
@@ -33,6 +33,7 @@
   'ctrl-alt-down': 'omnisharp-atom:navigate-down'
   'ctrl-.': 'omnisharp-atom:get-code-actions'
   'alt-enter': 'omnisharp-atom:get-code-actions'
+  'ctrl-shift-t': 'omnisharp-atom:find-symbols'
```

Comment:
@nosami  - is that not the same as ctrl on mac? I can't keep up with all the modifier keys they've added (but you only need one mouse button!).